icons/copy.svg

```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="9" y="9" width="13" height="13" rx="2" ry="2" />
-  <path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1" />
+  <path d="M4 16c-1.1 0-2-.9-2-2V4c0-1.1.9-2 2-2h10c1.1 0 2 .9 2 2" />
```

Comment:
I'm not sure with this change. there is more code added.
I think we can better use the old rect and path.

Reply:
Yeah, Illustrator seems reluctant to output `rect`s. I’ll just code these manually instead…